components/camel-mongodb/src/main/java/org/apache/camel/component/mongodb/conf/MongoConfiguration.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.mongodb.conf;
+
+import java.util.Map;
+
+public class MongoConfiguration {
```

Comment:
If this class is used on for extension, maybe it would be better to rename it.

Reply:
This doesn't seem to be resolved.

Reply:
@oscerd , can you please check again, I can see the change applied with the last commit, class now named differently.